### PR TITLE
Added Mailtrain and Slimta

### PR DESCRIPTION
**Pull request by madhugb**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -219,6 +219,7 @@ _MTAs / SMTP servers_
 * [Postfix](http://www.postfix.org/) - Fast, easy to administer, and secure Sendmail replacement. `IPL` `C`
 * [Qmail](http://www.qmail.org/top.html) - Secure Sendmail replacement. `CC0`
 * [Sendmail](http://www.sendmail.com/sm/open_source/) - Message transfer agent (MTA).
+ * [Slimta](http://slimta.org) - Mail Transfer Library built on Python. ([Source Code](https://github.com/slimta/python-slimta)) `MIT License` `Python`
```
Comment (nodiscc) [inline README.md]: wrong indentation (1 extra whitespace)

**Comment by nodiscc** [comment]
Thanks, there's just this indentation level problem. Can you fix it?

File: README.md (hunk 1 above)
Comment (madhugb) [inline README.md]: Done! Thanks.

File: README.md [hunk 2]
```diff
@@ -242,6 +243,7 @@ _Mailing lists servers._
 _Mass mailing software - one message to many recipients_
 
   * [phpList](https://phplist.org) - Newsletter and email marketing with advanced management of subscribers, bounces, and plugins. ([Source Code](https://github.com/phpList/)) `AGPLv3` `PHP`
+   * [Mailtrain](https://mailtrain.org/) - self hosted newsletter application built on Node.js (v5+) and MySQL (v5.5+ or MariaDB). ([Source Code](https://github.com/andris9/mailtrain)) `GPL-V3.0` `Node.js`
```
Comment (nodiscc) [inline README.md]: We missed this one. Same indentation problem.

Comment (madhugb) [inline README.md]: Fixed. Just noticed it. I guess my terminal generates one extra space (zsh)! Sorry for the trouble

File: README.md [hunk 3]
```diff
@@ -242,6 +243,7 @@ _Mailing lists servers._
 _Mass mailing software - one message to many recipients_
 
   * [phpList](https://phplist.org) - Newsletter and email marketing with advanced management of subscribers, bounces, and plugins. ([Source Code](https://github.com/phpList/)) `AGPLv3` `PHP`
+  * [Mailtrain](https://mailtrain.org/) - self hosted newsletter application built on Node.js (v5+) and MySQL (v5.5+ or MariaDB). ([Source Code](https://github.com/andris9/mailtrain)) `GPL-V3.0` `Node.js`
```
Comment (nodiscc) [inline README.md]: Please order the section alphabetically. I would be appreciated if you can squash your commits.

**Comment by nodiscc** [comment]
Squashed and merged in https://github.com/Kickball/awesome-selfhosted/commit/5d0f04983791b6deb81f829a4f61a49548589be7

**Pull request closed without merging**